kleinanzeigen.py

```diff
@@ -417,14 +422,27 @@ def session_create(config):
     log.info("Creating session")
 
     options = Options()
+    # for chromium driver
+    # add this option otherwise login is blocked when using selenium 
+    # https://stackoverflow.com/a/60649888/256002
+    # options.add_argument("disable-blink-features=AutomationControlled")
+
     if config.get('headless', False) is True:
         log.info("Headless mode")
         options.add_argument("--headless")
 
     if config.get('webdriver_enabled') is False:
         options.set_preference("dom.webdriver.enabled", False)
```

Comment:
Setting `dom.webdriver.enabled` to `False` here was working for me without issues. What do the new options below do? Also `dom.webdriver.enabled` is now set twice with different conditions.

Reply:
I have rechecked it, true. Removed unneeded directives and added a fix for python 3.6 and older with date formatting instead

Reply:
@neon-dev could you please merge? I'd like to dockerize the project in my next pull request

Reply:
I'm just a past contributor so I don't have write access, sorry. @donwayo has to merge this